app/assets/stylesheets/pageflow/themes/_base.scss

```diff
@@ -192,3 +200,56 @@ section.delayed_text_fade_in_short {
     font-size: 18px;
   }
 }
+
+
+//// Styling specific for the text page @Joost: Ruby gem TODO
+
+// A wrapper around image (centered text)
+.page .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .page .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .js .page .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .js .page .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .js .page:first-child .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .js .page:first-child .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .page:first-child .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image, .page:first-child .text_page .scroller.inline_text_position_left > div .contentInnerWrapper .contentText .inline_image,
```

Comment:
Ik heb deze uit Google dev tools gekopieerd. Ik dacht dat jij alleen de css, zonder selectors, zou kopieren en op de goede plek in de  Gem zou zetten? Zo niet, dubbele kunnen weg.